streamlog/streamlog

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# Print the Field System log to stdout
+set -euo pipefail
+
+if [[ -z ${FS_DISPLAY_SERVER:-} ]]; then
+    if ! fsserver status > /dev/null; then echo fsserver not running; exit 1 ; fi
+    if ! fsserver fs status > /dev/null; then echo fs not running; exit 1; fi
+    FS_SERVER_URL_BASE=$(awk '/FS_SERVER_URL_BASE/ {gsub(/"/, "", $3); print $3}' /usr2/fs/include/params.h)
+    exec ssub "$FS_SERVER_URL_BASE/log"
```

Comment:
For pre-beta1, replace this line with:

exec ssub -s -w ipc:///var/run/fsserver/windows/fs/{pub,rep}

Reply:
As above. An compelling reason to backport it?

Reply:
I see no reason to backport it, but there is a station with a need. They can use a local copy with the change until they can update. I thought it might be good to record the change needed. Maybe the update notice would be a better place.